models/junos/security_policies_global/junos_security_policies_global.yml

```diff
@@ -0,0 +1,213 @@
+        choices:
+          - deny-all
+          - permit-all
+      policy_rematch:
+        description:
+          - Enable the device to reevaluate an active session when its associated security policy is modified. The session remains open if it still matches the policy that allowed the session initially.
+        type: dict
+        suboptions:
+          extensive:
+            description:
+              - When a policy is modified or deleted, extensive option checks if any suitable policy permit to keep these sessions alive.
+            type: bool
+      policy_stats:
+        description:
+          - Configure systemwide policies statistics.
+        type: dict
+        suboptions:
+          system_wide:
+            description:
+              - Configure systemwide policies statistics.s
+            type: bool
+      pre_id_default_policy:
+        description:
+          - During the initial policy lookup phase, which occurs prior to a dynamic application being identified, if there are multiple policies present in the potential policy list, the SRX Series device applies the default security policy until a more explicit match has occurred. Configures default policy actions that occur prior to dynamic application identification (AppID).
+        type: dict
+        suboptions:
+          policy_action:
+            description:
+              - Specifies the policy action that has to be taken when the packet matches the criteria.
+            type: dict
+            suboptions:
+              log:
+                description:
+                  - Specifies the log details at session close time and session initialization time.
+                choices:
+                  - session-init
+                  - session-close
+              session-timeout:
+                description:
+                  - When you update a session, the session timeout is configured, which specifies the session timeout details in seconds.
+                type: dict
+                suboptions:
+                  icmp:
+                    description:
+                      - Timeout value for ICMP sessions (seconds)
+                    type: int
+                  icmp6:
+                    description:
+                      - Timeout value for ICMP6 sessions (seconds)
+                    type: int
+                  ospf:
+                    description:
+                      - Timeout value for OSPF sessions (seconds)
+                    type: int
+                  others:
+                    description:
+                      - Timeout value for other sessions (seconds)
+                    type: int
+                  tcp:
+                    description:
+                      - Timeout value for TCP sessions (seconds)
+                    type: int
+                  udp:
+                    description:
+                      - Timeout value for UDP sessions (seconds)
+                    type: int
+      stateful_firewall_rule_set:
```

Comment:
from which vsrx version we are getting this attribute ?

Reply:
This attribute is introduced in Junos OS 19.3R2.

https://www.juniper.net/documentation/us/en/software/junos/interfaces-next-gen-services/topics/ref/statement/statefu-firewall-rule-set-edit-services-usf.html